.circleci/install-content-server.sh

```diff
@@ -3,6 +3,10 @@
 DIR=$(dirname "$0")
 
 if grep -e "fxa-content-server" -e 'all' $DIR/../packages/test.list; then
+  CLOUD_SDK_REPO="cloud-sdk-$(grep VERSION_CODENAME /etc/os-release | cut -d '=' -f 2)"
+  echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+  curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+  sudo apt-get update -y && sudo apt-get install google-cloud-sdk -y
```

Comment:
I wonder if `sudo apt-get update ` will slow things down on the install even more :\

Reply:
It seemed to run pretty quick on circleci when I was watching it, took maybe 2 seconds. Granted that's in the ideal case where the network is behaving itself.

Reply:
thanks for checking!